### PR TITLE
Data-Model

### PR DESCRIPTION
**Pull request by Itswali**
Create a new app. Give it a meaningful name.
    Initialize your project with Git.
    Make sure that your project has Postgres database set up.
    Build your project schema.
        Create and run the necessary migration files.
        Table and column names should match the ERD diagram.
            note: photo for users table should be a link to an image
        Foreign keys should be included.
            Pay attention that in posts table the column holding the foreign key to the users table should be named author_id
        All columns that are foreign keys should have a corresponding index.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,24 +1,172 @@
...
+
+I would like to thank microverse inc. for helping me in my professional carreer.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LICENSE -->
+
+## 📝 License <a id="license"></a>
+
+This project is [MIT](./MIT.md) licensed.
```
Comment (DuaneDave) [inline README.md]: Please resolve the URL for the link here so it links properly to a license file in order to make your documentation complete.

**Review by DuaneDave — CHANGES_REQUESTED** [review]
Hi @Itswali 👋 , 

Good job so far on this project!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
✔️ Migrations are correctly created
✔️ Correct workflow in use


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
N/A

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by CollinsTatang — APPROVED** [review]
# Approved 👏  
### **Highlights 🚀 :heavy_check_mark: :heavy_check_mark:** 

- The repo looks professional.:heavy_check_mark:
- Git/GitHub flow best practices have been followed:heavy_check_mark:
- You have descriptive PR:heavy_check_mark:
- You meets the changes requested by the previous reviewer :heavy_check_mark: 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
### **Congratulations!** 🎉 
image



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag @CollinsTatang in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md (hunk 1 above)
Comment (Itswali) [reply README.md]: Location for the MIT Lincense file has been updated. 

Thanks For Pointing out the error.

**Pull request merged**